content/docs/2.9/scalers/pulsar.md

```diff
@@ -28,22 +29,42 @@ triggers:
 - `subscription` - Name of the topic subscription
 - `msgBacklogThreshold` - Average target value to trigger scaling actions. (default: 10)
 - `activationMsgBacklogThreshold` - Target value for activating the scaler. Learn more about activation [here](./../concepts/scaling-deployments.md#activating-and-scaling-thresholds).(Default: `0`, Optional)
+- `authModes` - a comma separated list of authentication modes to use. Default `""`. Valid options are `bearer`, `tls`, and `basic`. Note that `tls,bearer` or `tls,basic` are valid combinations and would indicate mutual TLS to secure the connection and then `bearer` or `basic` headers should be added to the HTTP request.
```

Comment:
```suggestion
- `authModes` - a comma separated list of authentication modes to use. (Values: `bearer`, `tls`,`basic`, Default: `""`, Optional, `tls,bearer` or `tls,basic` are valid combinations and would indicate mutual TLS to secure the connection and then `bearer` or `basic` headers should be added to the HTTP request)
```

Reply:
@JorTurFer Shouldn't the values/default be at the end?

Reply:
I have checked it, the style is 

- name - Description. (Values: x, y, z, Default: y, Optional, Extra Info)

So I think that my suggestion is correct